recall_server/voter/migrations/0016_alter_voter_password.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 5.0.6 on 2024-09-10 12:23
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("voter", "0015_alter_voter_password"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="voter",
+            name="password",
+            field=models.CharField(
+                default="pbkdf2_sha256$720000$24Unccq3zoKbk0KMJ3ZmSN$lspF4gisHl8sY4hu4A2qM2stPTWnQ14UwZcTqdyA/e8=",
```

Comment:
I did not include that, I was getting a merge conflict from that specific file. I thought is was because I had used User instead of AUTH_USER_MODEL as is in settings..changed that but this error persisted. I am not sure of its source but no there is absolutely no reason to have a default password,it's bad if anything.

Reply:
I think the biggest problem here is the migrations. I think we should gitignore them. I found it difficult to update the models with the current migrations.  I had to remove all the migrations and makemigrations again. That's when the changes applied. But don't worry about it.  I'll figure it out.

Reply:
Yes, issue came up on migrations.

Reply:
Okay. I'll try to fix it and let you know my progress via the discussion that I started